test/helper.rb

```diff
@@ -1,4 +1,15 @@
+require 'vcr'
+
 if ENV['TRAVIS']
   require 'coveralls'
   Coveralls.wear!
 end
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'test/cassettes'
+  config.hook_into :webmock
+  config.filter_sensitive_data('<AUTH_KEY>') { ENV['RENTLINX_AUTH_KEY'] }
```

Comment:
In the future, the auth key will be changing with some regularity, so you won't know it ahead of time when you're running the tests. Not sure what the solution is but it's something we should consider.